packages/presentation/src/components/Channels.svelte

```diff
@@ -15,66 +15,74 @@
 -->
 <script lang="ts">
   import type { Channel, ChannelProvider } from '@anticrm/contact'
-  import type { AttachedData, Doc, Ref } from '@anticrm/core'
+  import type { Doc, Ref, Timestamp } from '@anticrm/core'
   import type { Asset, IntlString } from '@anticrm/platform'
   import type { AnyComponent } from '@anticrm/ui'
   import { CircleButton, Tooltip } from '@anticrm/ui'
-  import { createEventDispatcher } from 'svelte'
-  import { getClient } from '..'
+  import { createEventDispatcher, getContext } from 'svelte'
+  import { Writable } from 'svelte/store'
   import { getChannelProviders } from '../utils'
   import ChannelsPopup from './ChannelsPopup.svelte'
 
-  export let value: AttachedData<Channel>[] | AttachedData<Channel> | null
+  export let value: Channel[] | Channel | null
   export let size: 'small' | 'medium' | 'large' | 'x-large' = 'large'
   export let reverse: boolean = false
   export let integrations: Set<Ref<Doc>> = new Set<Ref<Doc>>()
+  const lastViews = getContext('lastViews') as Writable<Map<Ref<Doc>, Timestamp>>
```

Comment:
Could we have this logic in separate notifications module API, so it could be possible to reuse with some APIs for different components?

Reply:
No, because each component may have custom logic, here as example check lastRead and if not exist items count

Reply:
At least 'lastViews' should be standardised as API.